.gitignore

```diff
@@ -0,0 +1,3 @@
+# .gitignore
+node_modules/
+.vscode/
```

Comment:
- [ ] Even though you have added the **vscode** to the ignore file, it still exists on your remote repo. Ensure that it doesn't surface remotely. This will be how you confirm that the file has been ignored.